cli/modules.h

```diff
@@ -3,6 +3,8 @@
  *  Distributed Under The MIT License
  */
 
+// Used for arguments parsing for REPL
+#include "thirdparty/argparse/argparse.h"
```

Comment:
`argparse` isn't a module of pocketlang (ie you cannot import argparse in pocketlang). So don't include this here. As for now include this in `main.c`